tests/bridge.test.js

```diff
@@ -14,11 +14,26 @@ describe('Bridge', () => {
     });
 
     it('should build the bridge contract with Ethers', () => {
-
         const provider = new ethers.JsonRpcProvider(networkUrl);
         const bridgeContract = new ethers.Contract(bridge.address, bridge.abi, provider);
         assert.equal(bridgeContract instanceof ethers.Contract, true);
-
     });
 
+    it('has all the expected functions with the expected properties', () => {
```

Comment:
This can be more complete. Let's make sure we have the proper format.

Assert that each element has a type, either `function` or `event`. Assert expected amount of functions/events.

Then assert that each element has the proper format. Functions have name, stateMutability, inputs, outputs. The last two should be lists.

Events have name, inputs. Inputs have indexed (true or false), name, type (check valid type).

And so on

Reply:
Done